Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'rake'
 group :development, :test do
   gem 'dotenv'
   gem 'roo'
-  gem 'atlas',    ref: 'a11c996', github: 'quintel/atlas'
+  gem 'atlas',    ref: '3631f56', github: 'quintel/atlas'
```

Comment:
The Atlas master branch ended up with a different commit ref after the merge: `98ce739`

Reply:
Fixed it 👍